ts/s3.ts

```diff
@@ -163,4 +164,16 @@ uploader2.send((err, data) => {
 
 });
 
-var endpoint: Endpoint = s3.endpoint;
\ No newline at end of file
+var endpoint: Endpoint = s3.endpoint;
+
+let presignedPost: PresignedPost = s3.createPresignedPost({
```

Comment:
Just to be safe, might be useful to also make sure we can access the expected fields (at least `url` and `fields`) on `presignedPost`.